runtime/manta/src/xcm_config.rs

```diff
@@ -313,6 +314,23 @@ parameter_types! {
     pub const MaxAssetsForTransfer: usize = 2;
 }
 
+const NATIVE_ASSET_IDX: MantaAssetId =
+    <MantaAssetConfig as AssetConfig<Runtime>>::NativeAssetId::get();
+
+match_types! {
+    pub type DisabledOutgoingAssets: impl Contains<CurrencyId> = {
```

Comment:
Seems you just defined it here, but no line to call it.

Reply:
yeah from earlier iteration, forgot to delete.